.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,27 @@
+name: Run Benchmarks
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  benchmark:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Run Go Benchmark Action
+        uses: patrickhuie19/benchmark-action@b46d0ce6c3b4bd423380f7d3c6abed389e38ea4a
```

Comment:
Why not putting it in here : https://github.com/smartcontractkit/.github/tree/main/actions ?

Reply:
Easier this way until we hit v1